### ISSUE TITLE
v0.12.1 missing sync dependency

### ISSUE THREAD

**Issue by bkutil** [piotrmurach/finite_machine#60]
### Describe the problem

The currently released version of finite_machine - [v0.12.1](https://rubygems.org/gems/finite_machine/versions/0.12.1) - raises `LoadError (cannot load such file -- sync)` on requiring the library.

This was fixed in (an unreleased) v0.13.0 [here](https://github.com/piotrmurach/finite_machine/commit/cc439cc81c3548fba930572e86b48349f94c04c0#diff-c309c558b7c24715645fbd446643342eR30).

It seems that the problem slipped through the CI, since `coveralls` also depends on `sync` [transitively](https://rubygems.org/gems/coveralls/versions/0.8.22/dependencies) through `tins`. But that gem is part of a "[metrics](https://github.com/piotrmurach/finite_machine/blob/173de0310ef2071c334b2ce8186c3e24a6e0ddb1/Gemfile#L11)" group in the `Gemfile` and not installed at runtime.

### Steps to reproduce the problem

Ensure that coveralls & sync gem are not installed.

```
git clone git@github.com:piotrmurach/finite_machine.git -b v0.12.1
cd finite_machine
bundle install --without=metrics
ruby -r./lib/finite_machine -e 'puts "works"'
```

### Actual behaviour

`LoadError` is raised due to `sync` dependency missing.

### Expected behaviour

No `LoadError` is raised.

### Describe your environment

* OS version: Linux dev 4.15.0-76-generic #86-Ubuntu SMP Fri Jan 17 17:24:28 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
* Ruby version: ruby 2.7.0p0 (2019-12-25 revision 647ee6f091)

**Comment by piotrmurach** [comment]
Hi Balázs,

Thanks for using the gem and reporting this.

The `sync` lib was part of Ruby standard lib until the version `2.6`. In Ruby `2.7` it was extracted out to a gem hence the loading issue. So it didn't 'slip' through CI as such but requires a new release. I will try to release the `v0.13.0` shortly.

**Comment by ahorek** [comment]
ping @piotrmurach 

do you have time to make a release?

**Comment by piotrmurach** [comment]
@ahorek Thanks for the updates. I'll release shortly - most likely tomorrow.

**Comment by piotrmurach** [comment]
Released `v0.13.0`.
